RCTAppleHealthKit/RCTAppleHealthKit+Methods_Activity.h

```diff
@@ -13,5 +13,6 @@
 - (void)activity_getActiveEnergyBurned:(NSDictionary *)input callback:(RCTResponseSenderBlock)callback;
 - (void)activity_getBasalEnergyBurned:(NSDictionary *)input callback:(RCTResponseSenderBlock)callback;
 - (void)activity_getAppleExerciseTime:(NSDictionary *)input callback:(RCTResponseSenderBlock)callback;
+- (void)activity_getAppleStandTime:(NSDictionary *)input callback:(RCTResponseSenderBlock)callback;
```

Comment:
Do we have the implementation of this method? I see it's signature being defined here at the headers, but I couldn't find the implementation of it under ` RCTAppleHealthKit+Methods_Activity.m`

We're expected to split the method into these two pieces, as a standard for Objective-C. Let me know if I'm missing something here 🤔